simphony/cuds/abc_modeling_engine.py

```diff
@@ -22,6 +25,31 @@ class ABCModelingEngine(object):  # pragma: no cover
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, *args, **kwargs):
+        """Initialize the engine.
+
+        Parameters
+        ----------
+        cuds:
+            The CUDS computational model object
+
+        Returns
+        -------
+
+        """
+        self._cuds = kwargs.get('cuds')
+        self._load_cuds()
+
+    # This class is supposed to be overridden, however it is not labeled
+    # abstract, in order to be backward-compatible.
```

Comment:
It might just be me: sometimes it is good to not be backward-compatible and force changes that are required.  And it seems to me this is the case here.